src/app/components/calendar/calendar.ts

```diff
@@ -243,6 +243,7 @@ export const CALENDAR_VALUE_ACCESSOR: any = {
                                                         [ngClass]="{ 'p-highlight': isSelected(date) && date.selectable, 'p-disabled': !date.selectable }"
                                                         (click)="onDateSelect($event, date)"
                                                         draggable="false"
+                                                        [attr.data-date]="formatDateKey(formatDateMetaToDate(date))"
```

Comment:
why to add attr.data-date ?

Reply:
To find the necessary cell for focus, you can refer to an example of its usage in row 2218. Another solution to this problem, albeit not ideal, is to use an ID. However, this approach may not be as effective.

Reply:
Thanks a lot we'll discuss whether to add new attribute or not and review the PR again upcoming week's release.